Gaijinizer patch/CommonEvents_336.txt

```diff
@@ -30,74 +33,78 @@ you feel?#MTLed
 \\C[2]正解をボクに提示できない限り
 ピンポーン♪は鳴らないからね？
 > CONTEXT: CommonEvents_336/11/Dialogue/enemy9
-I think I understand, but. \\C[2]Unless you can present
-the correct answer to me. Ping pong♪It won't
-sound, right?#MTLed
+I think you know this, but \\C[2]unless you can present
+the correct answer to me, that correct answer
+"ding" won't ring out.#Ron
 > END STRING
```

Comment:
Fitting in with the comment on the previous set of words, let's change this a tad:
>_**I think you know this already, but \\\C[2]unless you
can say the correct answer to me, that correct
answer "ding" won't ring out.**_